docs/content/reference/resource-schema/core-schema/container-schema/snippets/container.bicep

```
@@ -113,3 +120,19 @@ resource db 'Applications.Link/mongoDatabases@2022-03-15-privatepreview' = {
     application: app.id
   }
 }
+
+resource backend 'Applications.Core/containers@2022-03-15-privatepreview' = {
+  name: 'backend'
+  location: location
+  properties: {
+    application: app.id
+    container: {
+      image: magpieimage
```

Let's use the same format as above

```suggestion
      image: 'registry/container:tag'
```